libs/src/services/solanaWeb3Manager/index.js

```diff
@@ -222,8 +222,17 @@ class SolanaWeb3Manager {
    */
   async getWAudioBalance (solanaAddress) {
     try {
-      const tokenAccount = await this.getAssociatedTokenAccountInfo(solanaAddress)
-      if (!tokenAccount) return null
+      let tokenAccount = await this.getAssociatedTokenAccountInfo(solanaAddress)
```

Comment:
looks great - let's change the name of this guy at some point (doesn't have to be now) to just `getTokenAccountInfo`, since it works for any token account